src/webui/webapplication.cpp

```diff
@@ -429,6 +433,7 @@ void WebApplication::configure()
             LogMsg(tr("Using custom Web UI. Location: \"%1\".").arg(m_rootFolder));
     }
 
+    const bool isFirstTimeLoadingLocale = m_currentLocale.isEmpty();
```

Comment:
Maybe just set m_currentLocale to "en"?
Or even better don't try to load any translation in this case and set isTranslationNeeded to false in appropriate place. Then we don't need any dummy "en" translation file.

Reply:
I've updated the commit.